src/modules/governance/DelegatedInfoPanel.tsx

```diff
@@ -0,0 +1,203 @@
+  user,
+  aavePower,
+  stkAavePower,
+  aaveDelegatee,
+  stkAaveDelegatee,
+  title,
+}) => {
+  const isAaveSelfDelegated = !aaveDelegatee || user === aaveDelegatee;
+  const isStkAaveSelfDelegated = !stkAaveDelegatee || user === stkAaveDelegatee;
+
+  if (isAaveSelfDelegated && isStkAaveSelfDelegated) return null;
+
+  return (
+    <Box sx={{ display: 'flex', flexDirection: 'column', mt: 6, mb: 2 }}>
+      <Typography typography="caption" sx={{ mb: 5 }} color="text.secondary">
+        <Trans>{title}</Trans>
+      </Typography>
+      <Box sx={{ display: 'flex', gap: 4, flexDirection: 'column' }}>
+        {aaveDelegatee === stkAaveDelegatee ? (
+          <Row
+            align="flex-start"
+            caption={
+              <ExternalUserDisplay
+                avatarProps={{ size: AvatarSize.XS }}
+                titleProps={{ variant: 'subheader1' }}
+                address={aaveDelegatee}
+              />
+            }
+          >
+            <Box sx={{ display: 'flex', flexDirection: 'column', alignItems: 'flex-end' }}>
+              <FormattedNumber
+                value={Number(aavePower) + Number(stkAavePower)}
+                variant="subheader1"
+              />
+              <Typography variant="helperText" color="text.secondary">
+                AAVE + stkAAVE
+              </Typography>
+            </Box>
+          </Row>
+        ) : (
+          <>
+            {!isAaveSelfDelegated && (
+              <Row
+                align="flex-start"
+                caption={
+                  <ExternalUserDisplay
+                    avatarProps={{ size: AvatarSize.XS }}
+                    titleProps={{ variant: 'subheader1' }}
+                    address={aaveDelegatee}
+                  />
+                }
+              >
+                <Box sx={{ display: 'flex', gap: 2, alignItems: 'center' }}>
+                  <TokenIcon symbol="AAVE" sx={{ width: 16, height: 16 }} />
+                  <FormattedNumber value={aavePower} variant="subheader1" />
+                </Box>
+              </Row>
+            )}
+            {!isStkAaveSelfDelegated && (
+              <Row
+                align="flex-start"
+                caption={
+                  <ExternalUserDisplay
+                    avatarProps={{ size: AvatarSize.XS }}
+                    titleProps={{ variant: 'subheader1' }}
+                    address={stkAaveDelegatee}
+                  />
+                }
+              >
+                <Box sx={{ display: 'flex', gap: 2, alignItems: 'center' }}>
+                  <TokenIcon symbol="stkAAVE" sx={{ width: 16, height: 16 }} />
+                  <FormattedNumber value={stkAavePower} variant="subheader1" />
+                </Box>
+              </Row>
+            )}
+          </>
+        )}
+      </Box>
+    </Box>
+  );
+};
+
+export const DelegatedInfoPanel = () => {
+  const powers = useVotingPower();
+  const {
+    daveTokens: { aave, stkAave },
```

Comment:
I think this is just an existing typo, but we should probably update `daveTokens` to be `aaveTokens`